### ISSUE TITLE
Feature request: Auto walk/run key

### ISSUE THREAD

**Issue by DamnBriggsy** [H-uru/Plasma#945]
A bind-able key that, when pressed, would move the avatar forward automatically. Would ideally respect the walk/run toggle.

**Comment by Hoikas** [comment]
Considering the game already has a "Go To Where I Click" feature that functions more like "turn to where I click", we should probably fix that feature to do what it says(TM).

**Comment by Deledrius** [comment]
Any time I visit the Cleft, I find myself wishing Uru had a run lock like every other MMO has had since 2000.  It's far less needed in other Ages, but even outside of the Cleft and Minkata it would have uses.

**Comment by TheScarFr** [comment]
This already exists in CC: The Caps key.

**Comment by dgelessus** [comment]
Caps Lock works in MOULa too, for making the "forward" key run instead of walk by default. But I think this issue is asking for a key to make the avatar walk/run completely automatically, without having to keep the "forward" key pressed all the time.

**Comment by TheScarFr** [comment]
Then I don't understand the issue...
Because you don't have to keep the Caps key pressed, it works as a toogle.

**Comment by dgelessus** [comment]
> Or do you meant walking/running without having to press any key or without using the mouse?

Yes, exactly. As described at the top of the issue: "A bind-able key that, when pressed, would move the avatar forward automatically."

> If so, I don't see the point of this feature; at least, not in this game...

That's okay, you don't have to use it! 😉 But it seems that at least a couple of people are interested in this feature. Another comment mentions the Cleft and Minkata as two places where it would be especially useful.
